@here/harp-mapview/lib/MapView.ts

```diff
@@ -511,6 +512,13 @@ export interface MapViewOptions {
      * Disable all fading animations for debugging and performance measurement.
      */
     disableFading?: boolean;
+
+    /**
+     * Set to `true` to allow rendering scene synchronously.
```

Comment:
I think we need more documentation here what are the impacts if this is set to true (e.g. update will do nothing except triggering an event)
Would be also good to have a dedicated example for that.